packages/e2e/test/stake-pool/StakePoolProvider.test.ts

```diff
@@ -0,0 +1,384 @@
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+/* eslint-disable sonarjs/cognitive-complexity */
+import * as envalid from 'envalid';
+import { Cardano, QueryStakePoolsArgs, StakePoolProvider } from '@cardano-sdk/core';
+import { logger } from '@cardano-sdk/util-dev';
+import { stakePoolHttpProvider } from '@cardano-sdk/cardano-services-client';
+
+const stringToRegExEqualsTo = (str: string) => `^${str.replace(/[$()*+.?[\\\]^{|}-]/g, '\\$&')}$`;
+
+describe('stake-pool queries', () => {
```

Comment:
```suggestion
describe('StakePoolProvider', () => {
```
Best to keep the top level `describe` aligned with the filename